src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -1096,6 +1096,12 @@ private static SyntaxNode GetExpansionTarget(SyntaxToken token)
                 return enclosingInitializer.Value;
```

Comment:
(I can't add a comment for a line not appearing in the diff, :cry:)
Earlier in this method is a `possibleLambdaExpression` local that doesn't appear to be used that should probably be removed.